.github/workflows/flake8.yml

```diff
@@ -13,4 +13,5 @@ jobs:
       - name: Setup environment
         run: pipenv sync --dev
       - name: Run Flake8
-        run: pipenv run flake8 --max-line-length 120
\ No newline at end of file
+        run: pipenv run flake8 --max-line-length 120
+        
```

Comment:
I don't think you needed to create a new commit for that, right?
You could add this change to your last commit using the `git commit --amend` command.

Reply:
@RawanAbuleil please refer to some of the discussions started on other PRs and on Slack regarding pushing fixes to an existing PR.

Reply:
alright , thanks

Reply:
Missing one new line at the end of the  yml file, other then that, all looks good!